app/services/idv/inherited_proofing/service_providers.rb

```diff
@@ -0,0 +1,11 @@
+module Idv
+  module InheritedProofing
+    # This module is temporary until InheritedProofing has a viable way
+    # of identifying/communicating identifying Service Providers
+    # modules
+    module ServiceProviders
+      VA = :va
+      SERVICE_PROVIDERS = %i[VA]
```

Comment:
this constant is never used, seems like everything checks against `VA` directly, I'd opt to remove this

Reply:
I didn't think Melissa was going to use that code so quickly :O, before I had a chance to use it to validate against what's being passed to `ServiceProviderServiceFactory::Factory::Initialze` and `ServiceProviderFormFactory::Factory::Initialze`

You want us to stuff that in real quick?

Reply:
Done, in this PR: https://github.com/18F/identity-idp/pull/7181